src/test/java/edu/usf/cutr/gtfsrtvalidator/test/feeds/TripUpdateFeedTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.usf.cutr.gtfsrtvalidator.test.feeds;
+
+import edu.usf.cutr.gtfsrtvalidator.FeedMessageTest;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.CheckTripId;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.StopTimeSequanceValidator;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.VehicleIdValidator;
+import edu.usf.cutr.gtfsrtvalidator.validation.gtfs.StopLocationTypeValidator;
+import org.junit.Test;
+
+
+public class TripUpdateFeedTest extends FeedMessageTest{
+    
+    public TripUpdateFeedTest() throws Exception{}
+    
+    @Test
+    public void testTripIdValidation(){
+        
+        CheckTripId tripIdValidator = new CheckTripId();
+        
+        // feedMessage already have a valid trip_id set in InitRequiredFeedFields() method
+        errors = tripIdValidator.validate(gtfsData, feedMessage.build());
+        assertEquals(0, errors.getOccurrenceList().size());
+        
+        tripDescriptor.setTripId("100");
+        tripUpdate.setTrip(tripDescriptor);
+        feedEntity.setTripUpdate(tripUpdate);
+        feedMessage.setEntity(0, feedEntity);
+        
+        errors = tripIdValidator.validate(gtfsData, feedMessage.build());
+        assertEquals(1, errors.getOccurrenceList().size());
+        
+        clearAndInitRequiredFeedFields();
+    }
+    
+    @Test
+    public void testStopSequenceValidation() {
+        StopTimeSequanceValidator stopSequenceValidator = new StopTimeSequanceValidator();
+        
+        //ordered stop sequence 1, 5
+        stopTimeUpdate.setStopSequence(1);
+        tripUpdate.addStopTimeUpdate(stopTimeUpdate);
+        stopTimeUpdate.setStopSequence(5);
```

Comment:
Yes - so it should look something like:

~~~
// Ordered stop sequence 1, 5
StopTimeUpdate stopTimeUpdate = GtfsRealtime.TripUpdate.StopTimeUpdate.newBuilder();
stopTimeUpdate.setStopSequence(1);
tripUpdate.addStopTimeUpdate(stopTimeUpdate);
stopTimeUpdate = GtfsRealtime.TripUpdate.StopTimeUpdate.newBuilder();
stopTimeUpdate.setStopSequence(5);
tripUpdate.addStopTimeUpdate(stopTimeUpdate);
~~~

And, you could delete the `stopTimeUpdate` declarations in the super class, if they aren't used elsewhere.  I think its fine to re-use some of the parent objects like TripUpdates and VehiclePositions, but for objects we're manipulating within the unit test I'd prefer to encapsulate them in the unit test method.